### ISSUE TITLE
Build test for the web doesn't include webassembly

### ISSUE THREAD

**Issue by temeddix** [cunarist/rinf#87]
https://github.com/cunarist/rust-in-flutter/blob/dbb8bbb59b0c03df74465f96866b40b64c721bf6/.github/workflows/ci.yaml#L70-L72

Building webassembly before web build test is needed like this:
```yaml
...
      - name: Build Web
        working-directory: example/
        run: |
          dart run rust_in_flutter wasm --release
          flutter build web
...
```

However, for some reason, `dart run rust_in_flutter wasm` fails on GitHub CI. This command works well on local, so the reason of this CI behavior is not clear yet.

https://github.com/cunarist/rust-in-flutter/actions/runs/5852524994/job/15864901475

**Comment by temeddix** [comment]
@bookshiyi It looks like there's a room for improvment here..! 😐

**Comment by bookshiyi** [comment]
@temeddix yeah! I have tried to add to my test branch and debugging now.

[ci.yaml](https://github.com/bookshiyi/rust-in-flutter/blob/github-actions/.github/workflows/ci.yaml)

**Comment by bookshiyi** [comment]
> @bookshiyi It looks like there's a room for improvment here..! 😐

      # For web target
      - name: Setup WASM Toolchain
        if: matrix.target == 'web'
        run: cargo install wasm-pack

I think this might be effective.

**Comment by temeddix** [comment]
Okay :)

Also, let me mention that `dart run rust_in_flutter wasm` command includes automatic installation of `wasm-pack` and nightly Rust, so we do not need(and shouldn't add) manual installation of `wasm-pack` or other web-related tools in the CI.
https://github.com/cunarist/rust-in-flutter/blob/dbb8bbb59b0c03df74465f96866b40b64c721bf6/bin/rust_in_flutter.dart#L144-L168

**Comment by bookshiyi** [comment]
Initially, I did indeed run only the command `dart run rust_in_flutter wasm`, but GitHub Actions prompted that wasm-pack was missing. That's why I tried to add the step for manually installing **wasm-pack**. 
Thank you for providing this crucial information, and I will continue to troubleshoot the issue.

**Comment by temeddix** [comment]
Happy to hear your progress on that!

**Comment by bookshiyi** [comment]
> Okay :)
> 
> Also, let me mention that `dart run rust_in_flutter wasm` command includes automatic installation of `wasm-pack` and nightly Rust, so we do not need(and shouldn't add) manual installation of `wasm-pack` or other web-related tools in the CI.
> 
> https://github.com/cunarist/rust-in-flutter/blob/dbb8bbb59b0c03df74465f96866b40b64c721bf6/bin/rust_in_flutter.dart#L144-L168

You are right, some strange issues have arisen here like your description. 

I misunderstood your description earlier, and now I am trying to debug.

**Comment by bookshiyi** [comment]
PR #89

**Comment by temeddix** [comment]
Fixed by #89
